tests/python/gaia-ui-tests/gaiatest/tests/functional/gallery/test_gallery_view.py

```diff
@@ -46,7 +46,7 @@ def test_gallery_view(self):
         self.assertEqual(self.device.screen_width, image.photo_toolbar_width)
 
         #  Unlock the screen so that it can be changed back to portrait mode
-        self.marionette.execute_script('window.screen.mozUnlockRotation')
+        self.marionette.execute_script('window.screen.mozUnlockOrientation()')
```

Comment:
Even with the comment, I don't really understand what this line does, and why we need it to change the orientation again.